views/pages/AdminPage.jsx

```diff
@@ -22,29 +21,22 @@ import SelectField from '@material-ui/core/Select'
 import MenuItem from '@material-ui/core/MenuItem'
 import Checkbox from '@material-ui/core/Checkbox'
 import IconButton from '@material-ui/core/IconButton'
-import AppBar from '@material-ui/core/AppBar'
-import Toolbar from '@material-ui/core/Toolbar'
 import Typography from '@material-ui/core/Typography'
 import NoSsr from '@material-ui/core/NoSsr'
 import Select from 'react-select'
 import TextField from '@material-ui/core/TextField'
 import Paper from '@material-ui/core/Paper'
 import Chip from '@material-ui/core/Chip'
-import { emphasize } from '@material-ui/core/styles/colorManipulator'
 
-import { compose } from 'redux'
-import { withStyles } from '@material-ui/core/styles'
-
-import Sidebar from './components/Sidebar'
-import getAvatar from './fe-utils/avatarUtil'
-import getCounts from './fe-utils/countUtil'
+import getAvatar from '../fe-utils/avatarUtil'
+import getCounts from '../fe-utils/countUtil'
 import {
   fetchSubjects,
   fetchStudiesAdmin,
   fetchUsers,
   updateUser,
-} from './fe-utils/fetchUtil'
-import basePathConfig from '../server/configs/basePathConfig'
+} from '../fe-utils/fetchUtil'
+import basePathConfig from '../../server/configs/basePathConfig'
```

Comment:
can we still access the server directory now that the front-end is rendering these pages?

Reply:
Thanks for this. It does work, when this page is refactored this would be removed based on the pattern we are using